modules/learn/pages/data/change-history.adoc

```diff
@@ -0,0 +1,245 @@
+See xref:rest-api:rest-bucket-create.adoc[Creating and Editing Buckets].
+
+* The CLI, using `couchbase-cli bucket-create` (to create a bucket) or `couchbase-cli bucket-edit` (to edit a bucket); specifying the parameter `enable-history-retention-by-default`.
+See xref:cli:cbcli/couchbase-cli-bucket-create.adoc[bucket-create] and xref:cli:cbcli/couchbase-cli-bucket-edit.adoc[bucket-edit].
+
+[#individual-collection-setting]
+=== Individual Collection-Setting
+
+Change history can be switched off or on for each individual collection, overriding the established, bucket-wide default.
+This setting can be made either when the collection is created, or subsequently, by editing the collection's configuration.
+
+The setting can be established with either:
+
+* The REST API, using `POST /pools/default/buckets/<bucket_name>/scopes/<scope_name>/collections` (to create a collection) or `PATCH /pools/default/buckets/<bucket_name>/scopes/<scope_name>/collections/<collection_name>` (to edit); specifying the parameter `history`.
+See xref:rest-api:creating-a-collection.adoc[Creating and Editing a Collection].
+
+* The CLI, using `couchbase-cli collection-manage` (to create or edit a collection); specifying the parameter `enable-history-retention`.
+See xref:cli:cbcli/couchbase-cli-collection-manage.adoc[collection-manage].
+
+=== Change-History Status and Content-Access
+
+The change-history status of each collection can be returned by means of the `cbstats` tool, with the arguments `collections` and `collection-details`.
+Output indicates a status of `true` (change history is _on_) or `false` (change history is _off_).
+See the xref:cli:cbstats/cbstats-collections.adoc[collections] and xref:cli:cbstats/cbstats-collections-details.adoc[collections-details] reference pages for xref:cli:cbstats/cbstats-intro.adoc[cbstats].
+
+To access the change history and examine its contents, use the xref:4.1@kafka-connector::index.adoc[Kafka 4.1 Connector].
+
+=== Change-History Statistics
+
+Statistics are provided to allow change history to be monitored and managed.
+
+==== Statistics Provided by cbstats
+
+The `cbstats` tool, using the `all` argument, returns statistics that include the following:
+
+* `ep_total_enqueued` and `ep_total_deduplicated`.
+These respectively provide the total number of checkpointed items, and the number of items discarded due to deduplication.
+The numbers correspond to items for the entire bucket, and include both active and replica vBuckets.
++
+If deduplication is switched off, `ep_total_enqueued` represents all checkpointed items.
+In such a case, if the bucket has one replica, and the value of `ep_total_enqueued` is _n_, the number of writes enqueued to the change history is  _n/2_.
++
+If deduplication is switched _on_, the statistic `ep_total_deduplicated` represents the number of items recently removed from checkpoints for the whole bucket, through deduplication.
```

Comment:
remove "recently" , this stat represents the current total deduplicated for the time the bucket has been "up"

Reply:
I've removed 'recently', and have written: the number of items removed from checkpoints for the whole bucket, through deduplication, while the bucket has been _up_.